api/konfig/builtinpluginconsts/namereference.go

```diff
@@ -3,8 +3,9 @@
 
 package builtinpluginconsts
 
+// LINT.IfChange
```

Comment:
Oh this is cool! Didn't know you could do this

Reply:
well, actually this is just a convention and is not supported by go/github yet. In the future, we can customize a check to presubmit or github Action that "if a LINT commented block is changed, the files listed in `thenChange` should be changed in the PR".